src/ExtendedXmlSerializer/IExtendedXmlSerializerConfig.cs

```diff
@@ -27,12 +27,14 @@ namespace ExtendedXmlSerialization
 {
     public interface IExtendedXmlSerializerConfig
     {
-        Type Type { get; }
-        int Version { get; set; }
+        Type Type { get; } // Consider making obsolete as logic is now contained within implementation.
+        int Version { get; set; } // Consider making getter only, defined by implementation.
```

Comment:
Just realized... if we remove it, then it is a breaking change.  Are you OK with that?  Anyone who has created an implementation that uses this property will break on the next get.